src/backend/distributed/commands/utility_hook.c

```diff
@@ -904,6 +902,8 @@ ProcessUtilityInternal(PlannedStmt *pstmt,
 		 */
 		CitusHasBeenLoaded(); /* lgtm[cpp/return-value-ignored] */
 	}
+
+	SkipConstraintValidation = oldSkipConstraintsValidationValue;
```

Comment:
@marcocitus Should we reset this using `set_config_option` too?

Reply:
Let's leave it as is then, this seems an issue that won't really happen in practice for our current usage.